release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -28,13 +28,375 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 ### Language Updates
 
-#### New Features
+#### New features
 
-#### Bug Fixes
+##### Support for the `spread` operator in the `list` constructor
+
+Introduced the `spread` operator support for the `list` constructor expression.
+
+If the `spread` operator in a `list` constructor expression is `...x`, then, `x` is expected to be a list (i.e., an array or a tuple). All the member values of the list that result from evaluating `x` are included in the list value being constructed.
+
+```ballerina
+import ballerina/io;
+
+public function fn() {
+    int[] a1 = [3, 4];
+    int[] v1 = [1, 2, ... a1];
+    io:println(v1); // [1, 2, 3, 4]
+
+    int[2] a2 = [6, 7];
+    int[] v2 = [1, 2, ... a1, 5, ... a2];
+    io:println(v2); // [1, 2, 3, 4, 5, 6, 7]
+
+    [int, string] t1 = [5, "s"];
+    any[] v3 = [ ... t1, "x"];
+    io:println(v3); // [5, "s", "x"]
+
+    [boolean, int...] t2 = [false, 4, 7];
+    [string, int, string, boolean, int...] v4 = ["x", ... t1, ... t2];
```

Comment:
```suggestion
    [string, int, string, boolean, int...] v4 = ["x", ...t1, ...t2];
```
We need to update in other places as well

Reply:
Thanks. Added a single suggestion to fix all the formatting below.